models/public/mosaic/Mosaic.md

```diff
@@ -0,0 +1,102 @@
+# mosaic
+
+## Use Case and High-Level Description
+
+The `mosaic` model is one of the style transfer models designed to mix the content of an image with the style of another image. The model uses the method described in [Perceptual Losses for Real-Time Style Transfer and Super-Resolution](https://arxiv.org/abs/1603.08155) along with [Instance Normalization](https://arxiv.org/pdf/1607.08022.pdf).
+
+
+## Example
+
+
+## Specification
+
+| Metric            | Value            |
+|-------------------|------------------|
+| Type              | Style Transfer   |
+| GFLOPs            | -                |
+| MParams           | -                |
+| Source framework  | ONNX\*           |
+
+## Accuracy
+
+Accuracy metrics are obtained on MS COCO val2017 dataset. Images were resized to input size.
+
+| Metric | Original model | Converted model (FP32) | Converted model (FP16) |
+| ------ | -------------- | ---------------------- | ---------------------- |
+| PSNR   | 12.03Db        | 12.03Db                | 12.04Db                |
+
+## Performance
+
+## Input
+
+### Original model
+
+Image, name - `input1`,  shape - `1,3,224,224`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Expected color order: RGB.
+
+### Converted model
+
+Image, name - `input1`,  shape - `1,3,224,224`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Expected color order: BGR.
+
+## Output
+
+### Original model
+
+NumPy float32 array, name - `output1`, shape - `1,3,224,224`, format is `B,C,H,W` where:
```

Comment:
```suggestion
Image, name - `output1`, shape - `1,3,224,224`, format is `B,C,H,W` where:
```

At least I assume they're images... Also, you need to specify the color order.

Reply:
Fixed, Color order added next line like other models

Reply:
I don't think you did? It still says "NumPy float32 array".

Reply:
It is outdated file, file was renamed to `fast-neural-style-mosaic-onnx.md`

Reply:
Hmm, okay, I must've been looking at the wrong diff.